sale_order_product_recommendation/wizards/sale_order_recommendation.py

```diff
@@ -46,6 +46,14 @@ class SaleOrderRecommendation(models.TransientModel):
     def _default_order_id(self):
         return self.env.context.get("active_id", False)
 
+    def _prepare_sale_order_line_domain(self, sales):
```

Comment:
I think this title is a bit misleading.

```suggestion
    def _prepare_sale_order_line_recommendations_domain(self, sales):
```